src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/validation/ValidationConfig.java

```diff
@@ -0,0 +1,15 @@
+package ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.validation;
+
+import javax.persistence.EntityManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ValidationConfig {
+  @Bean
+  public OldStateFetcher oldStateFetcher(EntityManager entityManager) {
+    EntityManager secondEntityManager =
+        entityManager.getEntityManagerFactory().createEntityManager();
+    return new OldStateFetcher(secondEntityManager);
```

Comment:
Warum braucht man dieses Bean?

Reply:
Im normalen EntityManager ist das Game schon geändert. Deshalb brauchen wir einen zweiten, mit dem wir den zustand auf der Datenbank herausfinden können. (Link ist in der commit message)

Reply:
Die Rolle des EM habe ich einigermassen verstanden, aber die Rolle des Beans nicht. Mir ist nicht klar, warum der EM in einem Bean erstellt wird? 

Übrigens: müsste der EM nicht auch irgendwann wieder gelöscht/geschlossen werden?